modules/querying/pages/data-types.adoc

```diff
@@ -596,16 +596,23 @@ When the query where the `FILE` was declared finishes running, the `FILE` conten
 ==== Define s3 file object
 The path should start with `s3://`, followed by the bucket name and the S3 path, e.g., `s3://bucket-name/queryoutput/output.csv`. During the execution of the query, content will be uploaded to the S3 bucket. Note that the S3 object cannot be modified or appended, if an S3 object with the same path already exists, it will be overwritten.
 
-==== Set S3 connection credentials
-To use an S3 path, ensure that the necessary permissions and configurations are properly set up to allow read/write access to the specified S3 bucket. Configure the S3 connection credentials using `gadmin config`:
-[source,bash]
+==== Using Session Parameters
+Use the following session parameters in GSQL:
+[source,gsql]
 ----
-gadmin config set GPE.QueryOutputS3AWSAccessKeyID YOUR_AWS_ACCESS_KEY_ID
-gadmin config set GPE.QueryOutputS3AWSSecretAccessKey YOUR_AWS_SECRET_ACCESS_KEY
-gadmin config apply -y
-gadmin restart gpe -y
+set query_timeout=7200000; 
+set s3_aws_access_key_id = <AWS_KEY_ID>;
+set s3_aws_secret_access_key = <AWS_ACCESS_KEY>;
 ----
 
+* `set query_timeout=7200000;` sets the query timeout (this is optional but can be adjusted based on your query's expected duration).
+* `set s3_aws_access_key_id = <AWS_KEY_ID>;` specifies your AWS access key ID.
+* `set s3_aws_secret_access_key = <AWS_ACCESS_KEY>;` specifies your AWS secret access key.
+
+These session parameters should be set within the GSQL Editor to enable read/write access to the specified S3 bucket for query results.
+
+For more information on exporting query data to S3, refer to this xref:graph-development:advanced-features:write2-s3.adoc[detailed guide].
```

Comment:
You are linking to Savanna (cloud) documentation. That may not be appropriate for a self-managed user.